plugins/core/hu.bme.mit.gamma.statechart.model/src/hu/bme/mit/gamma/statechart/derivedfeatures/StatechartModelDerivedFeatures.java

```diff
@@ -1071,6 +1080,10 @@ public static Interface getContainingInterface(Event event) {
 		return ecoreUtil.getContainerOfType(event, Interface.class);
 	}
 	
+	public static Interface getContainingInterface(InterfaceParameterReferenceExpression expression) {
```

Comment:
This could be generalized to support future extensions automatically:
`public static Interface getContainingInterface(EObject object)`